living_documentation_regime/action_inputs.py

```diff
@@ -68,7 +68,7 @@ def get_liv_doc_regime() -> bool:
         Getter of the LivDoc regime switch.
         @return: True if LivDoc regime is enabled, False otherwise.
         """
-        return get_action_input(LIV_DOC_REGIME, "false").lower() == "true"
+        return get_action_input(Regime.LIV_DOC_REGIME, "false").lower() == "true"
```

Comment:
Why not call value() here and provide expected string into get_action_input instead of its refactoring?

Reply:
Good approach, changed in: TBD.